presentation/chapters/fr-FR/overview.chapter

```diff
@@ -25,7 +25,7 @@
 ## Développement
 
 -   Toujours avec un grand projet (ex: Servo)
--   Une adoption très tôt des releases régulière, depecrations et RFC processus
+-   Une adoption très tôt avec des releases régulières, et des obsolescences et RFC processus
```

Comment:
I think we should come back to the english meaning here, I don't exacly know what we want to say there.

Reply:
Basically, 
 This sentence "Early adoption of regular releases, deprecations and an RFC process" explains that by using a Request By Comment(RFC) approach it enables easy deprecations and making regular releases easier I think...

Reply:
What is your understanding of this one ? @Natim

Reply:
It plays to that the Rust project has adopted these practices very early. We had regular releases since 2010 (every 2 months back then), have had a deprecation system in the language before the 1.0 release and even before the 1.0 release, we had the RFC.

I hope that clarifies things.

Reply:
yep sure !

Thank you very much @skade